ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb

```diff
@@ -56,7 +56,7 @@
         "id": "-VL5avdUJ7-V"
       },
       "source": [
-        "### Install AI Platform (Unified) SDK\n",
+        "### Install Vertex AI SDK\n",
```

Comment:
"Vertex AI SDK" should be "Vertex SDK for Python" throughout these notebooks.

Reply:
Thanks, will do that.

Reply:
Done